suite/runsuite_wrapper.pl

```diff
@@ -249,6 +249,7 @@
                 'code_api|tool.drcacheoff.windows-invar' => 1, # i#6599
                 'code_api|tool.drcacheoff.invariant_checker' => 1, # i#6599
```

Comment:
(This comment belongs on line 242)

We should consider putting these "These are from the long suite" ones all under `$is_long`.  I don't think we had that variable available when these were first added, but we have it now.

Reply:
OTOH it could be that these aren't run in the shortened 32-bit windows suite so they might fail in a short suite if present.

Reply:
I had a similar thought during my review: should we have a separate ignore list for long suite tests. But would that be useful? Would we want to ignore a test in one suite but not the other? Initially I thought no, but maybe you think we should?

Reply:
Comment race:

> OTOH it could be that these aren't run in the shortened 32-bit windows suite so they might fail in a short suite if present.

Right. So maybe it's better to not gate it with `$is_long`.

Reply:
We do have some under $is_long today.  I think if we have the same test in both and it only fails in the long (can happen; remember the merge queue seems to be a little different env and things fail there but not in regular pre-merge) we'd want it in the separate long list.

Reply:
Unless a test is run differently !is_long vs is_long (is this one?) then I think it's a bug in the test harness and/or test if it fails in one and not the other. And if the test is run differently then it's really a different test - giving it the same name !long vs long will be a source of confusion.
Our test harness is already complex enough, and in need of some TLC (eg, #6607).
I would advocate for not trying to have a separate section for is_long.
Eg, moving a test from one to the other means having to update the flaky list - a test is either flaky or it isn't.
And if developers hack the test scripts to do experiments, remembering to update the flaky list is yet another bit of administrivia to have to deal with.

Reply:
Like I said, we have seen tests fail one Github Actions setup consistently and pass in another, and it's the same test.  So this can happen with different environments, when it's the exact same test on our side.